packaging/osx/buildpkg.sh

```diff
@@ -4,9 +4,17 @@ set -e
 
 umask 0022
 
-PKGFILE=$1
-VERSION=$2
-RELEASE=$3
+if [ "$#" -ne 2 ] ; then
+    echo "Usage: $0 <BUILDDIR> <SRCDIR>"
+    exit 1
+fi
+
+BUILDDIR="$1"
+SRCDIR="$2"
+
```

Comment:
can we throw an example/comment here to make the expected values obvious? I assume, BUILDDIR is a reference to the cmake build output directory, and that SRCDIR is the actual source code. Is that correct?

Reply:
Done, thanks